Various GTK 3.14 Fixes

gnunn1 opened the pull request:
Here are some changes to fix various 3.14 fixes, many of them are based on looking what you did for Paper which works well in 3.14 and applying the same to Orchis.

snwh commented:
Sure, thanks. I'll have to create the missing assets first before merging this pull request.

gnunn1 commented:
Just to be clear, the pull request only references assets in Orchis, I modified the CSS I extracted from Paper to remove any styles that referenced Paper assets that are not in Orchis. If I didn't do this UI elements disappeared when the application lost focus and various GTK errors were shown in the terminal due to assets not being resolveable. I'm not artistic hence creating the additional assets was not an option for me.

I'm not sure if what I did is the best approach but it does seem to be working fine with the gtk3-widget-factory and a few other applications I tested with. This is my first attempt at working with gtk themes so I'll defer to your expertise on this.

snwh commented:
Yes, that's why I said I have to make them before I merge this otherwise it will be built and distributed without them, buy the automagic build services. ;)

gnunn1 commented:
Just an FYI, I was little too aggressive in copying Paper CSS into buttons.css and there was a small issue where a fonts would be in white when hovering in certain lists like the tweak tool and the GEdit file drop down. I've fixed it in my forked repo, you can either reject this pull request and I'll submit a new one when the assets have been updated or I can just submit another pull request for the update, let me know what you prefer.
